superset-frontend/src/explore/components/controls/DndColumnSelectControl/utils/optionSelector.ts

```diff
@@ -19,26 +19,26 @@
 import { ColumnMeta } from '@superset-ui/chart-controls';
 
 export class OptionSelector {
```

Comment:
https://github.com/apache/superset/pull/13210/files#r583272323

Here's one way to make this generic so that adhoc columns/metrics can also be selected. But of course you can also make sure each adhoc item have a unique id/label and use is as the option key.